src/ebuckets.c

```diff
@@ -2250,6 +2333,42 @@ int ebucketsTest(int argc, char **argv, int flags) {
         }
     }
 
+    TEST("list - item defragmentation") {
```

Comment:
I think it is better to merge the two tests of list and rax into one test
such that it will make a loop, starting from 1 item up-to EB_LIST_MAX_ITEMS*3 in `ebuckets`.

Reply:
done with [`08c4276` (#13229)](https://github.com/redis/redis/pull/13229/commits/08c4276e491db80e91e95db706fd1e845ab12d51)